### PR TITLE
chore: Update Wallet disconnect SVG

### PR DESCRIPTION
**Pull request by cpcramer**
**What changed? Why?**
https://linear.app/coinbase/issue/BOE-326/[wallet]-update-the-disconnect-icon-to-an-updated-icon

We are upgrading to a slightly larger SVG image. 
Storybook:
Screenshot 2024-08-19 at 9 29 32 PM

Screenshot 2024-08-20 at 9 22 17 PM

<table>
  <tr>
    <th>Before</th>
    <th>After</th>
  </tr>
  <tr>
    <td>

Screenshot 2024-08-19 at 4 52 51 PM

</td>
    <td>
Screenshot 2024-08-19 at 4 52 59 PM

   </td>
  </tr>
</table>

<table>
  <tr>
    <th>Before</th>
    <th>After</th>
  </tr>
  <tr>
    <td>
Screenshot 2024-08-19 at 4 53 18 PM

</td>
    <td>
Screenshot 2024-08-19 at 4 53 27 PM

   </td>
  </tr>
</table>
**Notes to reviewers**

**How has it been tested?**

### CODE REVIEW

File: src/internal/svg/disconnectSvg.tsx [hunk 1]
```diff
@@ -6,16 +6,16 @@ export const disconnectSvg = (
     aria-label="ock-disconnect-svg"
     width="100%"
     height="100%"
-    viewBox="0 0 20 20"
+    viewBox="0 0 16 20"
```
Comment (Zizzamia) [inline src/internal/svg/disconnectSvg.tsx:9]: Do we have a story for this component on Storybook?

Comment (cpcramer) [reply src/internal/svg/disconnectSvg.tsx:9]: No, we do not have a storybook for any of our internal SVGs. Should I try to make one?

Comment (cpcramer) [reply src/internal/svg/disconnectSvg.tsx:9]: I have a Fix Storybook task on Linear but it's getting deprioritized for tasks such as fix landing page and the upcoming Swap Component changes https://linear.app/coinbase/issue/BOE-313/fix-storybook

Comment (Zizzamia) [reply src/internal/svg/disconnectSvg.tsx:9]: My comment was more about. Go make a storybook page for every component you touch :) 

So in this case you touch this component, let's have a story for this.

Comment (cpcramer) [reply src/internal/svg/disconnectSvg.tsx:9]: TIL you can add a story for an SVG! Added!

**Comment by cpcramer** [comment]
TODO: we are testing the Svg used inside the component add the <WalletDropdownDisconnect component as story

File: src/internal/svg/disconnectSvg.tsx (hunk 1 above)
Comment (abcrane123) [inline src/internal/svg/disconnectSvg.tsx:9]: curious why you updated this to 16 here ? are the other SVGs the same dimensions ?

File: src/internal/svg/disconnectSvg.stories.ts [hunk 2]
```diff
@@ -0,0 +1,23 @@
+import type { Meta, StoryObj } from '@storybook/react';
+import React from 'react';
+import { disconnectSvg } from './disconnectSvg';
+
+const SvgWrapper: React.FC = () =>
```
Comment (abcrane123) [inline src/internal/svg/disconnectSvg.stories.ts:5]: nice

File: src/internal/svg/disconnectSvg.tsx (hunk 1 above)
Comment (cpcramer) [reply src/internal/svg/disconnectSvg.tsx:9]: `viewBox 0 0 16 20 ` was specified in this new Svg - when downloading the image and displaying the code. 

The image is off center when using `0 0 20 20`.

**Review by Zizzamia — APPROVED** [review]

**Pull request merged**